sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-BCAG.e2e-spec.ts

```diff
@@ -0,0 +1,75 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
```

Comment:
Are we not replicating the tests for the other program year ? to keep it consistent with fulltime workflow e2e tests.

Reply:
I was following fulltime, I have only done for `2023-2024` for FT, 
![image](https://github.com/bcgov/SIMS/assets/77353155/f02d623a-1642-43aa-8062-44d7782aa9fb)

Reply:
I see. @andrewsignori-aot  I assume it is because the award eligibility is very less likely to be modified during program year ?

Reply:
@dheepak-aot @ann-aot I guess that it is just a gap from the previous implementation. I do believe that we should have per-program year E2E tests also.